src/funcx_common/messagepack/message_types/result.py

```diff
@@ -20,20 +21,8 @@ class Result(Message):
     task_id: uuid.UUID
     data: str
     error_details: t.Optional[ResultErrorDetails]
-    exec_start_ms: t.Optional[int]
-    exec_end_ms: t.Optional[int]
-    # storage for the computed property, "private" (meaning it will not appear in
-    # serialized data)
-    _exec_duration_ms: t.Optional[int] = None
+    statuses: t.Optional[t.List[t.Tuple[int, TaskState, ActorName]]]
```

Comment:
I'm not a huge fan of the big compound primitive type being used here, especially how it isn't obvious that the `int` member in the innermost tuple is intended to be a timestamp.

Would introducing a new type to replace the innermost tuple, maybe even just a `ValueTuple`, work here? One drawback I can think of would be extra noise in the logs - I'm not sure what the right solution to that would be, though, if we're trying to avoid that, so I'm definitely open to discussion there